plugins/cue_points/base/lib/kCuePointManager.php

```diff
@@ -1,11 +1,80 @@
 <?php
+
+class KAMFData
+{
+	public $pts;
+	public $ts;
+
+};
+
 /**
  * @package plugins.cuePoint
  */
-class kCuePointManager implements kObjectDeletedEventConsumer, kObjectChangedEventConsumer, kObjectAddedEventConsumer, kObjectCreatedEventConsumer, kObjectReplacedEventConsumer
+class kCuePointManager implements kBatchJobStatusEventConsumer, kObjectDeletedEventConsumer, kObjectChangedEventConsumer, kObjectAddedEventConsumer, kObjectReplacedEventConsumer
 {
 	const MAX_CUE_POINTS_TO_COPY_TO_VOD = 100;
 	const MAX_CUE_POINTS_TO_COPY_TO_CLIP = 1000;
+	const CUE_POINT_TIME_EPSILON = 1;
+
+	/* (non-PHPdoc)
+ 	 * @see kBatchJobStatusEventConsumer::updatedJob()
+ 	 */
+	public function updatedJob(BatchJob $dbBatchJob)
+	{
+		KalturaLog::debug('in kCuePointManager.updatedJob - jobID:' . $dbBatchJob->getId() . ' job type: ' . $dbBatchJob->getJobType()  . ' JobStatus= ' . $dbBatchJob->getStatus());
```

Comment:
Class and method are already added automatically to the log, also, each event consumer already logged with it's related object, please remove all redundant logs.